[2025-05-21T16:19:50.297+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-21T16:19:50.324+0000] {taskinstance.py:2631} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2025-01-02T00:00:00+00:00 [queued]>
[2025-05-21T16:19:50.343+0000] {taskinstance.py:2631} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2025-01-02T00:00:00+00:00 [queued]>
[2025-05-21T16:19:50.345+0000] {taskinstance.py:2884} INFO - Starting attempt 1 of 6
[2025-05-21T16:19:50.369+0000] {taskinstance.py:2907} INFO - Executing <Task(_PythonDecoratedOperator): get_name> on 2025-01-02 00:00:00+00:00
[2025-05-21T16:19:50.377+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=10826) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-21T16:19:50.384+0000] {standard_task_runner.py:72} INFO - Started process 10832 to run task
[2025-05-21T16:19:50.382+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_api_v02', 'get_name', 'scheduled__2025-01-02T00:00:00+00:00', '--job-id', '3500', '--raw', '--subdir', 'DAGS_FOLDER/dar_with_taskflow_api.py', '--cfg-path', '/tmp/tmpi_yp120o']
[2025-05-21T16:19:50.386+0000] {standard_task_runner.py:105} INFO - Job 3500: Subtask get_name
[2025-05-21T16:19:50.472+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_api_v02.get_name scheduled__2025-01-02T00:00:00+00:00 [running]> on host 2032228d86df
[2025-05-21T16:19:50.722+0000] {taskinstance.py:3157} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_api_v02' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2025-01-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-01-02T00:00:00+00:00'
[2025-05-21T16:19:50.724+0000] {taskinstance.py:740} INFO - ::endgroup::
[2025-05-21T16:19:50.729+0000] {python.py:240} INFO - Done. Returned value was: {'first_name': 'Jerry', 'last_name': 'Fridman'}
[2025-05-21T16:19:50.855+0000] {taskinstance.py:349} INFO - ::group::Post task execution logs
[2025-05-21T16:19:50.856+0000] {taskinstance.py:361} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_api_v02, task_id=get_name, run_id=scheduled__2025-01-02T00:00:00+00:00, execution_date=20250102T000000, start_date=20250521T161950, end_date=20250521T161950
[2025-05-21T16:19:51.020+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-21T16:19:51.087+0000] {taskinstance.py:3924} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-21T16:19:51.135+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
